cmake/FindLibBcc.cmake

```diff
@@ -38,9 +38,9 @@ find_library (LIBBCC_LIBRARIES
     ENV LIBRARY_PATH
     ENV LD_LIBRARY_PATH)
 
-find_library (LIBBPF_LIBRARY_STATIC
+find_library (LIBBCC_BPF_LIBRARY_STATIC
```

Comment:
Why does this need to change?

Reply:
NVM, i realized because this build is so old it was before bcc started vendoring libbpf